src/apiCall.js

```diff
@@ -0,0 +1,21 @@
+const BASE_URL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/';
+const gameId = 'fI1tffmB55k9l8dBQbMp';
+
+export const addScore = async (name, score) => {
+  await fetch(`${BASE_URL}${gameId}/scores`, {
+    method: 'POST',
+    body: JSON.stringify({
+      user: name,
+      score,
+    }),
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  }).then((res) => res.json());
+};
+
+export const getScore = async () => {
+  let data;
+  await fetch(`${BASE_URL}${gameId}/scores`).then((res) => res.json()).then((res) => { data = res; });
+  return data;
+};
```

Comment:
- [ ] This function is working as expected, but it would be nice to make the scores displayed on the screen when the page is loaded for the first time, so anyone can see all the scores without pressing the refresh button.
- [ ] Also, it would be nice to display the scores after you submit a new score, so no need to press the refresh button as well.
So please, work on these issues.